src/request-pipeline/header-transforms.js

```diff
@@ -92,6 +92,9 @@ var responseTransforms = {
     'x-content-security-policy-report-only': skip,
     'x-webkit-csp':                          skip,
 
+    // NOTE: Even if we were not able to authorize we should prevent native browser credentials window.
```

Comment:
// NOTE: Even if we are not able to authorize a user, we should prevent showing the native credentials window.